packages/contracts-bedrock/test/cannon/MIPS2.t.sol

```diff
@@ -1130,7 +1130,7 @@ contract MIPS2_Test is CommonTest {
     )
         public
     {
-        vm.assume(_wakeup != sys.FUTEX_EMPTY_ADDR);
+        _wakeup = uint32(_bound(_wakeup, 0, sys.FUTEX_EMPTY_ADDR - 1));
```

Comment:
We should instead check if the value is equal to this and do an addition. Otherwise we are bounding the result and not getting the full range of possible values.

Reply:
FUTEX_EMPTY_ADDRESS is type(uint32).max that's why I used bound here

Reply:
Ah ok